source/extensions/common/wasm/plugin.h

```diff
@@ -15,15 +15,21 @@ namespace Extensions {
 namespace Common {
 namespace Wasm {
 
+// clang-format off
+using EnvironmentVariableMap = std::unordered_map<std::string, std::string>;
```

Comment:
Nit: `absl::flat_hash_map`?

Reply:
The reason why this is not absl is that we have decided not to use absl as a dependency in proxy-wasm-cpp-host which is supposed to be used in other proxies in theory.